### DISCUSSION TITLE
SCA summary: Benchmark for Windows audit: Score less than 80% (71)

### DISCUSSION THREAD

**Discussion by isaacgolding** [Security-Onion-Solutions/securityonion#8019] [Unsupported Versions] [answered]
So I get this alerts today on my SO install  .  I handed it over to cases so I can examine it further.   When I go into cases and dig through all of the information this topic shows I get no further information anywhere.   
(screen caps attached)
 I know from prior dealings with Wazuh and some other systems I explored in the past that this is likely from the single wazuh-agent I have installed on 1 windows computer and that it was performing a pci-compliance scan or something along those lines.   (see url for SCA info) [](https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/what-is-it.html)

What I'm trying to figure out is if this message simply got parsed to the point that all the important bits are gone, or is something missing in the wazuh system on SO that is in place in a "default" wazuh install?

![Capture2](https://user-images.githubusercontent.com/104205381/170714126-1d3e9ffc-d08a-4fff-9b10-2148bbf8c2a0.JPG)
![Capture1](https://user-images.githubusercontent.com/104205381/170714130-1163ccba-2bdf-46ff-bbb3-e6d02d9f6102.JPG)

**Comment by dougburks** [comment, marked as answer]
It looks like you escalated from the aggregated view rather than drilling into the aggregation and then escalating the individual item. From https://docs.securityonion.net/en/2.3/cases.html#creating-a-new-case:
> Alternatively, if you find events of interest in [Alerts](https://docs.securityonion.net/en/2.3/alerts.html#alerts), [Dashboards](https://docs.securityonion.net/en/2.3/dashboards.html#dashboards), or [Hunt](https://docs.securityonion.net/en/2.3/hunt.html#hunt), you can escalate directly to Cases using the escalate button (blue triangle with exclamation point). Clicking the escalate button will escalate the data from the row as it is displayed. This means that if you’re looking at an aggregated view, you will get limited details in the resulting escalated case. If you want more details to be included in the case, then first drill into the aggregation and escalate one of the individual items in that aggregation.
